ckan/templates/organization/search_rebuild.html

```diff
@@ -0,0 +1,42 @@
+{# (canada fork only): background search index rebuilding #}
+{#  TODO: upstream contrib!! #}
+{% extends "organization/edit_base.html" %}
+
+{% block subtitle %}{{ _('Rebuild Search Index') }} {{ g.template_title_delimiter }} {{ super() }}{% endblock %}
+
+{% block primary_content_inner %}
+  {% if job_info %}
+    <div class="row">
+      <div class="col-md-12">
+        <div data-module="progress-bar" data-module-endpoint="{{ h.url_for('util.search_rebuild_progress', entity_id=group_dict.get('id')) }}"></div>
+      </div>
+    </div>
+    <hr />
+  {% endif %}
+  <form method="post">
+    {% if 'csrf_input' in h %}
+      {{ h.csrf_input() }}
+    {% endif %}
```

Comment:
`if 'csrf_input' in h` is only for extensions that need to support old versions of ckan

Reply:
yeah, this was for our 2.9 fork as we don't have it yet. But won't be putting it in the upstream contrib or in our 2.10 branch